komoju-php/komoju-php/lib/komoju/KomojuApi.php

```diff
@@ -105,4 +109,19 @@ private function post($uri, $payload)
 
         return $decoded;
     }
+
+    private function headers()
+    {
+        $result = [
+            'Content-Type: application/json',
+            "komoju-via: {$this->via}",
+        ];
+
+        $waf_token = get_option('komoju_woocommerce_waf_staging_token');
+        if ($waf_token) {
+            $result[] = "Cookie: waf_staging_token=$waf_token";
+        }
+
+        return $result;
+    }
```

Comment:
This change is actually not strictly related. I should've put it in another PR, but I wanted to test with staging, which is impossible without the waf staging token.